e2e-tests/playwright/support/ui/components/channels/sidebar_right.ts

```diff
@@ -29,7 +29,8 @@ export default class ChannelsSidebarRight {
         this.scheduledDraftChannelInfoMessageText = container.locator('span:has-text("Message scheduled for")');
         this.rhsPostBody = container.locator('.post-message__text');
         this.postCreate = new components.ChannelsPostCreate(container.getByTestId('comment-create'), true);
-        this.closeButton = container.locator('#rhsCloseButton');
```

Comment:
the ID is customized by RHS callers. The search results for example do not use this ID.